CHANGES

```diff
@@ -7,6 +7,8 @@
 
 # Error checking default changed to runtime (used to be all).
 
+# Make MPI_COMM_SELF the default communicator for MPI_ERRORS_RETURN.
```

Comment:
That's imprecise. How about - " Use the errorhandler bound to MPI_COMM_SELF as the default error handler"?

Reply:
Done